packages/graphql-playground-electron/webpack.config.js

```diff
@@ -17,8 +16,9 @@ if (!fs.existsSync(appEntrypoint)) {
 
 module.exports = {
   devtool: 'cheap-module-eval-source-map',
+  mode: 'development',
   entry: './src/renderer',
-  target: 'electron',
+  target: 'electron-main',
```

Comment:
I think we want `electron-render` here. `electron-main` is the node runtime, where as `electron-render` is chromium